api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -508,28 +517,28 @@ public void openmrsDateFormat_shouldNotAllowDatesWithInvalidDaysOrMonths() throw
 		try {
 			SimpleDateFormat sdf = OpenmrsUtil.getDateFormat(new Locale("en", "GB"));
 			sdf.parse("1/13/2001");
-			Assert.fail("Date with invalid month should throw exception.");
+			fail("Date with invalid month should throw exception.");
```

Comment:
Is switching from Assert.fail to fail part of the changes required for the ticket? Or did you just change it accidentally? :)